README.md

```diff
@@ -64,137 +88,159 @@ class { 'wsus_client':
 }
 ~~~
 
-### Enabling Status Server
+To automatically serve clients the correct updates, WSUS needs to know which updates the client has already installed. Clients can report this to a Windows Update status server as defined by the `WUStatusServer` policy, which must have the same value as the `WUServer` policy to be valid for automatic updates. For details, see the [Microsoft TechNet documentation][TechNet].
```

Comment:
This is an incorrect statement, the reporting server is for audit tracking not to determine what updates to send to a given server.  These are determined upon update checking as part of the `detection frequency` parameter.